app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -520,6 +529,81 @@ private void showInfoDialog() {
         infoDialog.create().show();
     }
 
+    private void showBalanceUpdateDialog() {
+        Context dialogContext = this;
+
+        AlertDialog.Builder updateDialog = new AlertDialog.Builder(this);
+        LinearLayout layout = new LinearLayout(this);
+        layout.setOrientation(LinearLayout.VERTICAL);
+
+
+        TextView updateTitleView = new TextView(this);
+        updateTitleView.setPadding(20, 20, 20, 20);
```

Comment:
I'm seeing a lot of padding, font size and colour changes. I suppose you based your UI mostly on the other dialog in the LoyaltyCardViewActivity? After it was changed to MaterialAlertDialogBuilder the padding there seems all wrong and does need fixing. If you still want more Hacktoberfest PRs, letting this wait and picking up #1074 could be a good thing as fixing that will hopefully tell you exactly how to best structure this dialog.

Reply:
Please let me know how the changed one looks. Unfortunately, I need to use a custom view to be able to dynamically update the balance number. I think, for static text the setMessage method would solve it all.